etna/insights/models.py

```diff
@@ -30,14 +32,25 @@ def get_context(self, request):
 
     content_panels = BasePage.content_panels + [
         FieldPanel("sub_heading"),
-        PageChooserPanel("featured_insight"),
         StreamFieldPanel("body"),
     ]
     promote_panels = BasePage.promote_panels + TeaserImageMixin.promote_panels
 
     subpage_types = ["insights.InsightsPage"]
 
 
+@register_snippet
+class InsightsPageTag(TaggedItemBase):
+    content_object = ParentalKey(
+        "insights.InsightsPage", on_delete=models.CASCADE, related_name="tagged_items"
+    )
+    free_tagging = False
+
+    class Meta:
+        verbose_name = "name"
+        verbose_name_plural = "slug"
```

Comment:
I can sense a bit of confusion here. These are labels for the model class. I would expect them to look something like:

```
    verbose_name = "insights tag"
    verbose_name_plural = "insights tags"
```

Reply:
Yes absolutely , done now